test/BaselineOutput/Common/LightGBMR/LightGBMReg-CV-generatedRegressionDataset-out.txt

```diff
@@ -35,10 +35,10 @@ Virtual memory usage(MB): %Number%
 [1] 'Loading data for LightGBM' started.
 [1] 'Loading data for LightGBM' finished in %Time%.
 [2] 'Training with LightGBM' started.
-[2] (%Time%)	Iteration: 50	Training-l2: 37.107605006517
+[2] (%Time%)	Iteration: 50	Training-rmse: 6.09160118577349
```

Comment:
>rmse [](start = 36, length = 4)

I am investigating the delta in these numbers. I believe it has to do with the metric type we were previously passing vs what is passed now. Previously we were setting the metric type to l2 which is incorrect for rmse. Now we are passing l2_root.

Reply:
Confirmed this is due to changing the code to use l2_root(aka rmse). 

---
In reply to: [266183353](https://github.com/dotnet/machinelearning/pull/2948#discussion_r266183353) [](ancestors = 266183353)